tests/integration/test_weights_loading.py

```diff
@@ -0,0 +1,37 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import pytest
+
+import torch
+from torchvision.models.mobilenet import model_urls
+from torch import hub
+from pytest_mock import MockerFixture
+
+from jinahub.image.encoder import ImageTorchEncoder
+
+
+@pytest.fixture()
+def mobilenet_weights(tmpdir: str) -> str:
+    weights_file = os.path.join(tmpdir, 'w.pth')
```

Comment:
why do u need this? mobilenet is downloaded autmoatically

Reply:
I use it for a test where I try to use pretrained weights via the `load_pretrained_from_path` parameter and check that they are loaded correctly.

Reply:
I should simulate the users behaviour in case he brings his own weights.